sql/education_schools(grade 1-12).sql

```diff
@@ -0,0 +1,267 @@
+--
+-- PostgreSQL database dump
+--
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+
+SET search_path = public, pg_catalog;
+
+
+ALTER TABLE IF EXISTS ONLY public.education_schools DROP CONSTRAINT IF EXISTS education_schools_pkey;
+DROP TABLE IF EXISTS public.education_schools;
+
+SET search_path = public, pg_catalog;
+
+SET default_tablespace = '';
+
+SET default_with_oids = false;
+
+--
+-- Name: education_schools; Type: TABLE; Schema: public; Tablespace:
+--
+
+CREATE TABLE education_schools(
+    geo_level character varying(15) NOT NULL,
+    total integer NOT NULL,
+    "school type" character varying(128) NOT NULL,
+    geo_code character varying(10) NOT NULL
+);
+
+
+--
+-- Data for Name: education_schools; Type: TABLE DATA; Schema: public
+--
+COPY education_schools( geo_level, total, "school type", geo_code) FROM stdin WITH DELIMITER ',';
+country,29035,Community schools,NP
+country,6566,Institutional Schools,NP
+country,35601,Total schools,NP

```

Comment:
A "total" row is not needed and should be removed. Please see my comment on the teachers data for an explanation why.